tests/integration.rs

```diff
@@ -7325,6 +7325,55 @@ mod common_parallel {
 
         handle_child_output(r, &output);
     }
+
+    #[test]
+    #[cfg(target_arch = "x86_64")]
+    fn test_nmi() {
+        let jammy = UbuntuDiskConfig::new(JAMMY_IMAGE_NAME.to_string());
+        let guest = Guest::new(Box::new(jammy));
+        let api_socket = temp_api_path(&guest.tmp_dir);
+        let event_path = temp_event_monitor_path(&guest.tmp_dir);
+
+        let kernel_path = direct_kernel_boot_path();
+        let cmd_line = format!("{} {}", DIRECT_KERNEL_BOOT_CMDLINE, "unknown_nmi_panic=1");
+
+        let mut cmd = GuestCommand::new(&guest);
+        cmd.args(["--cpus", "boot=1"])
```

Comment:
As per your comment, we cannot have more than 1 guest vcpu if we have inject the NMI?

Reply:
The problem has been fixed now.
The integration test here keeps 1 vcpu just is the same with other test case.

Reply:
You can test with multiple CPUs if you think that will be helpful.

Reply:
Okay. Updated from 1 to 4 vcpus.